### PR TITLE
Fixes #123 --  feat. remove mentioning of old, deprecated and discountinued m…

### LINKED ISSUE #123: Remove migration script from Migration Guide page
The [Migration Guide](https://www.openmage.org/magento-lts/migration-guide.html) page still mentions the migration script which was deprecated and no longer works.

### Related

- https://github.com/OpenMage/magento-lts/issues/3361
- https://github.com/OpenMage/magento-lts/issues/3343

### PR DESCRIPTION
**Pull request by pquerner**
…igration script, fix some minor typos, add livereload to jekyll

### CODE REVIEW

File: magento-lts/migration-guide.html [hunk 1]
```diff
@@ -299,10 +206,11 @@ <h2 class="migration-method__title">
                                 <img class="migration-switcher__header-icon migration-switcher__header-icon--plus" src="/assets/svg/plus.svg" />
                                 <img class="migration-switcher__header-icon migration-switcher__header-icon--minus" src="/assets/svg/minus.svg" />
                                 <p class="migration-switcher__header-text">
-                                    5. Rebase with latest Openamge branch (1.9.4.x currently)
+                                    5. Rebase with OpenMage branch (here: 1.9.4.x)
```
Comment (fballiano) [inline magento-lts/migration-guide.html:209]:
```suggestion
                                    5. Rebase with OpenMage branch (here: main)
```

File: magento-lts/migration-guide.html [hunk 2]
```diff
@@ -299,10 +206,11 @@ <h2 class="migration-method__title">
...
                                 </p>
                             </div>
                             <div class="migration-switcher__info">
+                                We go in more detail about other versions <a href="/supported-versions.html">here</a>.<br><br>
                                 <pre><code>git rebase -X theirs upstream/1.9.4.x</code></pre>
```
Comment (fballiano) [inline magento-lts/migration-guide.html:214]:
```suggestion
                                <pre><code>git rebase -X theirs upstream/main</code></pre>
```

**Review by colinmollenhour — APPROVED** [review]
Looks good, thanks!

**Review by fballiano — APPROVED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -9,4 +9,4 @@ services:
     volumes:
       - ./:/app
     working_dir: /app
-    command: "jekyll serve --host=0.0.0.0 --force_polling"
+    command: "jekyll serve --livereload --host=0.0.0.0 --force_polling"
```
Comment (fballiano) [inline docker-compose.yml:12]: hope this doesn't create problems to the live environment, I've no idea :-D

Comment (pquerner) [reply docker-compose.yml:12]: Uuuuh.. does live use docker?!

Comment (fballiano) [reply docker-compose.yml:12]: I really don't know, probably not

Comment (pquerner) [reply docker-compose.yml:12]: I thought this is just a 1:1 page directly to OpenMage.github.io as static pages - and the domain simply points to that github.io thingy. (Changes are live now aswell, so it was okay. And livereload is nice during developing. Sorry for changing too much out of scope of that ticked)

Comment (colinmollenhour) [reply docker-compose.yml:12]: Live definitely doesn't use Docker, I just added the docker file for local development.

**Pull request merged**